### PR TITLE
Add low bandwidth mode

### PR DESCRIPTION
**Pull request by pnts-se-whereby**
### Description

**Summary:**

**Related Issue:**

### Checklist

-   [ ] My code follows the project's coding standards.
-   [ ] Prefixed the PR title and commit messages with the service or package name
-   [ ] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [ ] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: packages/media/src/webrtc/VegaRtcManager/index.ts [hunk 1]
```diff
@@ -782,17 +782,19 @@ export default class VegaRtcManager implements RtcManager {
...
                     stopTracks: false,
-                    ...getMediaSettings("video", false, this._features),
+                    ...mediaSettings,
                     appData: {
                         streamId: OUTBOUND_CAM_OUTBOUND_STREAM_ID,
                         sourceClientId: this._selfId,
                         screenShare: false,
                         source: "webcam",
                         paused: currentPaused,
+                        numSimulcastLayers: mediaSettings?.encodings?.length || 0,
```
Comment (geirbakke) [inline packages/media/src/webrtc/VegaRtcManager/index.ts:797]: Is this needed for setMaxSpatialLayer to not fail - or just to get the -1?
Will it work for SVC? (do we then specify multiple encodings?)

**Review by geirbakke — APPROVED** [review]
LGTM 🍰  but left a comment to consider.

File: packages/media/src/webrtc/VegaRtcManager/index.ts (hunk 1 above)
Comment (geirbakke) [reply packages/media/src/webrtc/VegaRtcManager/index.ts:797]: what happens if you user number.MAX_VALUE?

Comment (pnts-se-whereby) [reply packages/media/src/webrtc/VegaRtcManager/index.ts]: Yes, that is easier and seems to work.

**Pull request merged**